definitions/ext-dell_compellent/summary_metrics.yml

```diff
@@ -0,0 +1,7 @@
+diskCount:
```

Comment:
as we talked on private the summary should be a copy of the golden metrics unless there's a reason for it to be different.

Reply:
unless I am missing it - the Disk Count definition is the same between summary/golden metrics @naxhh